src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -3050,6 +3051,35 @@ public void Escape(object parameter)
             BackgroundPreviewViewModel.RefreshState();
         }
 
+        /// <summary>
+        /// Checking if a custom Group style has been updated, if so it will update the styling of the existing groups
+        /// </summary>
+        /// <param name="originalCustomGroupStyles"></param>
+        public void CheckCustomGroupStylesChanges(List<GroupStyleItem> originalCustomGroupStyles)
+        {
+            foreach (var originalCustomGroupStyle in originalCustomGroupStyles)
+            {
+                var currentCustomGroupStyle = PreferenceSettings.GroupStyleItemsList.Where(
+                    groupStyle => groupStyle.GroupStyleId != Guid.Empty && groupStyle.GroupStyleId == originalCustomGroupStyle.GroupStyleId).FirstOrDefault();
+
+                if (currentCustomGroupStyle != null)
+                {
+                    if (originalCustomGroupStyle.HexColorString != currentCustomGroupStyle.HexColorString
+                        || originalCustomGroupStyle.FontSize != currentCustomGroupStyle.FontSize)
+                    {
+                        foreach (var annotation in CurrentSpaceViewModel.Annotations)
+                        {
+                            if (annotation.GroupStyle == currentCustomGroupStyle.GroupStyleId)
```

Comment:
I would compare both style id and name?

Reply:
I don't think so because the name is not editable, the user only has the chance to change the color and font size, the id is only for keep the reference of the applied style.